src/concepts/external_data_integration.md

```diff
@@ -0,0 +1,75 @@
+# External data integration
```

Comment:
I second that.

Reply:
Agreed.
I think we have most of these alternate paths though because Fivetran wasn't an option back then. Specifically for stripe this would have potentially been easier: https://fivetran.com/docs/applications/stripe